pkg/local_object_storage/shard/container.go

```diff
@@ -39,3 +45,86 @@ func (s *Shard) ContainerSize(prm ContainerSizePrm) (ContainerSizeRes, error) {
 		size: size,
 	}, nil
 }
+
+// DeleteContainer deletes any information related to the container
+// including:
+// - Metabase;
+// - Blobstor;
+// - Pilorama (if configured);
+// - Write-cache (if configured).
+func (s *Shard) DeleteContainer(_ context.Context, cID cid.ID) error {
+	s.m.RLock()
+	defer s.m.RUnlock()
+
+	var wg errgroup.Group
+
+	objs, err := s.metaBase.GetContainer(cID)
+	if err != nil {
+		return fmt.Errorf("fetching '%s' container objects: %w", cID, err)
```

Comment:
no need to add func input to err context (same below)
```suggestion
		return fmt.Errorf("fetching container objects: %w", err)
```